hs/Scripts.hs

```diff
@@ -71,7 +71,9 @@ graphScripts = do
          "/static/js/graph/sidebar/focus_descriptions.js",
          "/static/js/common/export/export.js"])
     H.script ! A.type_ "text/jsx" ! A.src "/static/js/common/react_modal.js.jsx" $ ""
-
+    H.script ! A.type_ "text/jsx" ! A.src "/static/js/graph/react_node.js.jsx" $ ""
```

Comment:
I'm not exactly sure what do do here.
@david-yz-liu Can you give some advice please?